packages/shared/src/permissions/permissionsClass.ts

```diff
@@ -17,6 +18,43 @@ export class Permissions {
     this.superAdmin = superAdmin;
   }
 
+  // This is a helper method to use on the frontend to determine whether or not to show certain UI elements
+  public canViewIdeaModal = (project?: string): boolean => {
+    return this.checkProjectFilterPermission(
+      {
+        projects: project ? [project] : [],
+      },
+      "createIdeas"
+    );
+  };
```

Comment:
@bryce-fitzsimons New pattern introduced based on your comment Friday. If we move forward with this pattern, I'll open a PR to add this pattern to all of the previously migrated permissions